modules/python/python-logger.c

```diff
@@ -67,7 +67,10 @@ PyObject *
 py_msg_debug(PyObject *obj, PyObject *args)
 {
   if (!debug_flag)
-    return Py_None;
+    {
+      Py_INCREF(Py_None);
```

Comment:
I do not really understand the ref here. Can you elaborate a bit please? Why is it not needed elsewhere?

Reply:
It fixes a bug. It might be needed elsewhere, but it could be handled in a separate PR.